content/2025-08-20-heapless-091.md

```diff
@@ -0,0 +1,114 @@
++++
+title = "Heapless v0.9.1 has been released!"
+date = 2025-08-20
+draft = false
+in_search_index = true
+template = "page.html"
++++
+
+# Heapless `0.9.1` released
+
+Almost 2 years after the last release, the [heapless](https://github.com/rust-embedded/heapless) crate has a new release. The first attempt at a `0.9.0` release was yanked due to including more breaking changes than intended.  This has been fixed, and `0.9.1`  has been released today.
```

Comment:
Sorry for the nitpick but could we please keep all the lines under 100 chars and break lines on the word boundary based on this limit rather than for starting a new sentence (in the same paragraph)? :pray: